pynamodb/tests/test_attributes.py

```diff
@@ -151,17 +151,31 @@ def test_utc_date_time_deserialize(self):
             attr.deserialize(tstamp.strftime(DATETIME_FORMAT)),
         )
 
-    def test_utc_date_time_deserialize_parse_args(self):
+    def test_dateutil_parser_fallback(self):
+        """
+        UTCDateTimeAttribute.deserialize
+        """
+        expected_value = datetime(2047, 1, 6, 8, 21, tzinfo=tzutc())
+        attr = UTCDateTimeAttribute()
+        self.assertEqual(
```

Comment:
It would help me a lot if you changed this to `assert expected_value == attr.deserialize('January 6, 2047 at 8:21:00AM UTC')` as I would like to move this towards pytest style unit tests. but if it feels weird to have two different styles, I can do it later.